### ISSUE TITLE
[question] how to disable inserting "(" when entering ")"?

### ISSUE THREAD

**Issue by ghost** [Fuco1/smartparens#517]
When I type the right brace ")", sometime the left brace "(" will be automatically inserted at somewhere. How can I disable the functionality?

I check the wiki, and only find one related control variable `sp-cancel-autoskip-on-backward-movement`. but sometime it still didn't work well.

Thanks a lot.

**Comment by Fuco1** [comment]
If it is inserted "somewhere" randomly that's probably a bug.  Try to find a repro case and share with us :)

**Comment by gygl09** [comment]
I believe it is not a bug. In deed, smartparens did a great job on matching braces.

I just don't like that functionality _personally_. Sometimes I type ")", then "(" is also inserted (for example, in the previous line). But I don't notice it.  Later, I change my mind, and I delete ")", but at that time "(" is left. So when I re-compile my program, I have to find that "(" to exclude the compilation error.

It is not convenient for me to find an example. If I got one, I would paste it here :)

I just would like to disable it. Now I set `sp-cancel-autoskip-on-backward-movement` to nil. Will it work?
